src/generator/resources/Vector.st

```diff
@@ -47,6 +53,91 @@ public class $vector/mutableType()$ extends $vector/baseType()$ {
         this($components:{x| other.get$x.proper$()}; separator = ", "$);
     }
 
+$if(componentType.discrete)$
+    /**
+     $components:{x| * @param $x$ the $x$ component}; separator = "\n"$
+     */
+    public $vector/mutableType()$($components:{x|float $x$}; separator = ", "$) {
+        this($components:{x|
+            DoubleMath.roundToInt($x$, RoundingMode.FLOOR)};
```

Comment:
Before you/we merge: in the `TeraMath` class of engine are methods for ceil- and floor-to-int (https://github.com/MovingBlocks/Terasology/blob/develop/engine/src/main/java/org/terasology/math/TeraMath.java#L749-L767). Do you plan to move them over later on, or should we generally use `DoubleMath`?